consumer/src/mode/types.rs

```diff
@@ -96,10 +96,7 @@ impl DecodedConsumerContext {
 
     /// This function fetches the current contract balance using the provider,
     /// since your contract does not expose a `balance()` function.
-    pub async fn fetch_contract_balance(
-        &self,
-        // event: &DecodedMessage,
-    ) -> Result<U256, ConsumerError> {
+    pub async fn fetch_contract_balance(&self) -> Result<U256, ConsumerError> {
```

Comment:
We should be getting contract balance for a specific block. Otherwise stats_hour will be off when reindexing